tests/Query/DbTestCase.php

```diff
@@ -44,4 +61,15 @@ public function assertDqlProducesSql($actualDql, $expectedSql, $params = []): vo
 
         $this->assertEquals($expectedSql, $actualSql);
     }
+
+    public static function ormVersionCompare(string $version): int
+    {
+        if (class_exists('\Doctrine\ORM\Version')) {
+            return Version::compare($version);
+        }
+
+        $installedVersion = InstalledVersions::getVersion('doctrine/orm');
+
+        return version_compare($installedVersion, $version);
+    }
```

Comment:
Please let's use feature detection instead.

Reply:
If I understand correctly, the original purpose of the author was to be able to handle changes to the SQL parenthesis rendered from DQL. I followed this logic where there is a difference between ORM2 and ORM3.

Reply:
Yes.